nix/vast/plugins/versions.nix

```diff
@@ -0,0 +1,6 @@
+{
+  compaction = "1.4.0";
+  inventory = "1.0.0";
+  matcher = "3.0.0";
+  netflow = "1.0.0";
+}
```

Comment:
Didn't we wanna get rid of these?

Reply:
That also needs changes to the C++ code for the version command. I hoped to do that in a follow up PR.